examples/local/indepth/main.go

```diff
@@ -162,6 +176,11 @@ func run(log logging.Logger, binaryPath string) error {
 	log.Info("updated network's nodes: %s", nodeNames)
 	log.Info("Network will run until you CTRL + C to exit...")
 	// Wait until done shutting down network after SIGINT/SIGTERM
-	<-closedOnShutdownCh
-	return nil
+	select {
+	case <-unexpectedNodeStopCh:
+		// Ctrl+C also kills childs before ANR does, so only use the event to stop, no notification
```

Comment:
Is this true?

Reply:
refer to previous comment/example

Reply:
The logic of this PR is that if a process dies before ANR signals it to die, it is considered an unexpected stop.
It does not matter if it dies by having received a signal, for having incorrect params, etc.
When Ctrl+C happens, all processes are signalled before ANR signals them. So it can happen that a process dies before ANR signals it. In relation to the examples, they shall assume that may get unexpected node failure messages after SIGINT (and, really, at this moment you just want to stop the network). Probably a function can be added to ANR to be notified of beign under SIGINT, so as to quickly cancel sending messages of this kind.
I modified the examples so they are faster in communicating and propagating the SIGINT event. 
I think the gain is more by having ANR provide the user this channel for all interesting cases, and ignore the channel for this pretty special case where the shell sends kill to all processes.

Reply:
The previous example answer the question related to the behaviour of Ctrl+C
Regardong the ignored unexpected stop, I modified the example